tests/main/ubuntu-core-classic/task.yaml

```diff
@@ -14,6 +14,9 @@ prepare: |
 
 restore: |
     rm -f /etc/sudoers.d/create-test
+    # Undo the change done by the classic snap.
+    # FIXME: https://github.com/snapcore/classic-snap/issues/32
+    mount devpts -t devpts /dev/pts -o remount,nosuid,noexec,relatime,gid=5,mode=620,ptmxmode=000
```

Comment:
I'd try to catch the bad state, causing it to fail once a fix is implemented:

```
# Undo the change done by the classic snap.
# FIXME: https://github.com/snapcore/classic-snap/issues/32 the check will fail once a fix is in place
grep '/ /dev/pts' /proc/self/mountinfo | MATCH " rw,mode=666,ptmxmode=666"
# restore the original state
mount devpts -t devpts /dev/pts -o remount,nosuid,noexec,relatime,gid=5,mode=620,ptmxmode=000
```

Reply:
I don't want to introduce another reason for our tests to fail. I'm looking at the associated bug and hopefully can fix both sides, rendering this workaround void.